config/routes.rb

```diff
@@ -9,7 +9,9 @@
 
   post 'login', to: 'session#create'
   delete 'logout', to: 'session#destroy'
-
+  
+  resources :agent_statuses, :defaults => { :format => :json }
```

Comment:
Don't use resources, as it creates a bunch of routes that are not needed for the project. Instead restrict the existing specified routes to JSON only.